src/auth/auth.controller.ts

```diff
@@ -65,6 +66,16 @@ export class AuthController {
     return this.authService.login(data);
   }
 
+  @ApiOperation({
+    summary: 'OAuth2.0 로그인 - 모바일 파트',
+    description: '유저 확인 후 회원가입/로그인',
+  })
+  @ApiResponse({ status: 200, description: '로그인 또는 회원가입 성공' })
+  @Post('/auth/mobile')
+  oauthMobileLogin(@Body() data: OauthMobileLoginDto) {
+    return this.authService.oauthMobileLogin(data);
+  }
+
```

Comment:
```suggestion
  @Public()
  @ApiOperation({
    summary: 'OAuth2.0 로그인 - 모바일 파트',
    description: '유저 확인 후 회원가입/로그인',
  })
  @ApiResponse({ status: 200, description: '로그인 또는 회원가입 성공' })
  @Post('/auth/mobile')
  oauthMobileLogin(@Body() data: OauthMobileLoginDto) {
    return this.authService.oauthMobileLogin(data);
  }
```
`@Public()`을 붙여야 token 검사를 안 해요